src/pyscript/__init__.py

```diff
@@ -7,13 +7,13 @@
 
 APPNAME = "pyscript"
 APPAUTHOR = "python"
-DEFAULT_CONFIG_FILENAME = "pyscript.json"
+DEFAULT_CONFIG_FILENAME = ".pyscriptconfig"
```

Comment:
Maybe a comment would help here to clear up the difference between the config file for the CLI itself, and the project config file? When I first looked at the code it took me a minute to see why DEFAULT_CONFIG_FILENAME didn't match the value in DEFAULT_CONFIG. Or maybe (although it is longer), always refer to them as either CLI_CONFIG_FILENAME, cli_config and PROJECT_... etc etc?